### PR TITLE
DBN-changes

### PR DESCRIPTION
**Pull request by palashahuja**

### CODE REVIEW

File: pgmpy/models/DynamicBayesianNetwork.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+        >>> moral_graph = dbn.moralize()
+        >>> moral_graph.edges()
+        [(('G', 0), ('I', 0)),
+        (('G', 0), ('D', 0)),
+        (('D', 1), ('I', 1)),
+        (('D', 1), ('G', 1)),
+        (('I', 0), ('D', 0)),
+        (('G', 1), ('I', 1))]
+        """
+        from pgmpy.base import UndirectedGraph
```
Comment (ankurankan) [inline pgmpy/models/DynamicBayesianNetwork.py:372]: @palashahuja If this doesn't cause a circular dependency, please put this on the top of the file.

**Comment by palashahuja** [comment]
@ankurankan The change that you reverted needs to be made because that's why the travis is failing

**Comment by ankurankan** [comment]
@palashahuja Will look into it.

**Comment by palashahuja** [comment]
@ankurankan Why did you merge this branch, what about the failing travis?

**Comment by ankurankan** [comment]
@palashahuja: I made some changes before merging..

**Pull request merged**